fem/fespace.hpp

```diff
@@ -330,6 +330,11 @@ class FiniteElementSpace
    virtual const Operator *GetProlongationMatrix() const
    { return GetConformingProlongation(); }
 
+   /// Return a prolongation operator that only acts on local dofs
+   /** The returned operator is owned by the FiniteElementSpace. */
+   virtual const Operator *GetLocalProlongationMatrix() const
+   { return GetConformingProlongation(); }
```

Comment:
It's not clear what this means in the context of a serial FiniteElementSpace. I suppose it's just the AMR P matrix, but perhaps the documentation should make clear that there is only a distinction in the parallel case?

Reply:
Done.